pkg/grpc/server.go

```diff
@@ -206,9 +208,9 @@ func (s *server) Close() error {
 	return nil
 }
 
-func (s *server) getMiddlewareOptions() []grpc.ServerOption {
-	intr := make([]grpc.UnaryServerInterceptor, 0, 6)
-	intrStream := make([]grpc.StreamServerInterceptor, 0, 5)
+func (s *server) getMiddlewareOptions() []grpcGo.ServerOption {
+	intr := make([]grpcGo.UnaryServerInterceptor, 0, 6)
```

Comment:
See my comment on the other thing. I don't think a variable would help because this should ideally be interpreted by the compiler. But I will add a comment to explain.